test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -41,12 +41,10 @@
 import org.jboss.builder.BuildContext;
 import org.jboss.builder.BuildStep;
 import org.junit.jupiter.api.extension.AfterEachCallback;
-import org.junit.jupiter.api.extension.BeforeAllCallback;
 import org.junit.jupiter.api.extension.BeforeEachCallback;
 import org.junit.jupiter.api.extension.ExtensionContext;
 import org.junit.jupiter.api.extension.TestInstanceFactory;
 import org.junit.jupiter.api.extension.TestInstanceFactoryContext;
-import org.junit.jupiter.api.extension.TestInstancePostProcessor;
 import org.junit.jupiter.api.extension.TestInstantiationException;
 import org.junit.platform.commons.JUnitException;
 import org.objectweb.asm.ClassReader;
```

Comment:
Could we have a test for this mojo just to be sure it "works"

Reply:
I'll have to figure out how to write one.

Reply:
Look how the tests of the main Maven plugin are written. Basically, you provide a project and run it. Then, you check that nothing unexpected happened.

Reply:
Added a simple test.